purchases/src/test/java/com/revenuecat/purchases/PurchaseParamsTest.kt

```diff
@@ -44,6 +46,23 @@ class PurchaseParamsTest {
         assertThat(purchaseProductParams.presentedOfferingIdentifier).isEqualTo(STUB_OFFERING_IDENTIFIER)
     }
 
+    @Test
+    fun `Initializing with TestStoreProduct throws error`() {
```

Comment:
Added this test to ensure that attempting to purchase a `TestStoreProduct` fails early.